lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -86,6 +97,17 @@ func listSubscribersHandler(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+
+	// Single-get subscriber by alias
+	sub, err := loadSubscriberByAlias(c, networkID)
+	if err != nil {
+		return makeErr(err)
+	}
+	if sub != nil {
+		return c.JSON(http.StatusOK, sub)
```

Comment:
if you want to mix the return type for the handler, you should specify the msidn-query version of the call as a new endpoint. E.g. see `/networks/{network_id}/policies/rules?view=full`. Otherwise generated client code will be incorrect the swaggerfile.